flutter_app/lib/main.dart

```diff
@@ -72,19 +76,25 @@ class _MyHomePageState extends State<MyHomePage> {
   TextEditingController _messageController = TextEditingController();
   Timer serverTimeTimer;
   Timer pingTimer;
+  String _username;
+  String _icon;
   bool isPlaying = false;
   bool connected = false;
   int videoDuration = 655550;
   List<UserMessage> userMessages = new List();
 
-
+  _MyHomePageState() {
+    _loadUsernameAndIcon();
```

Comment:
does this get called again when visiting this widget from the settings? if so, perfect, bc then the settings update will be broadcast automatically